src/screens/Distribution/DistributionCard.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+import React, { PureComponent } from "react";
+import { View, StyleSheet } from "react-native";
+import { BigNumber } from "bignumber.js";
+import type {
+  CryptoCurrency,
+  TokenCurrency,
+} from "@ledgerhq/live-common/lib/types/currencies";
+import colors from "../../colors";
+import LText from "../../components/LText";
+import CurrencyIcon from "../../components/CurrencyIcon";
+import CurrencyUnitValue from "../../components/CurrencyUnitValue";
+import ProgressBar from "../../components/ProgressBar";
+import CounterValue from "../../components/CounterValue";
+import CurrencyRate from "../../components/CurrencyRate";
+
+export type DistributionItem = {
+  currency: (CryptoCurrency | TokenCurrency) & { color: string },
```

Comment:
I prefer we stick to 

`currency: CryptoCurrency | TokenCurrency`

because this is getting ugly